.github/workflows/pre_merge.yaml

```diff
@@ -84,6 +84,38 @@ jobs:
           curl -Os https://uploader.codecov.io/latest/linux/codecov
           chmod +x codecov
           ./codecov -t ${{ secrets.CODECOV_TOKEN }} --sha $COMMIT_ID -U $HTTP_PROXY -f .tox/coverage_unit-test-${{ matrix.tox-env }}.xml -F ${{ matrix.tox-env }}
+  Intense-Unit-Test:
```

Comment:
Perhaps that can be removed one we downsize our models portfolio. 
cc @kprokofi @ashwinvaidya17

Reply:
Maybe we can also remove tox as it is slower. @samet-akcay saw some speedup by using direct pytest call. Also, he is working on a new design for testing.